website/docs/d/fabric_datastore_vsphere.html.markdown

```diff
@@ -0,0 +1,58 @@
+---
+layout: "vra"
+page_title: "VMware vRealize Automation: fabric_datastore_vsphere"
+description: |-
+  Provides a data lookup for vSphere fabric datastores.
+---
+
+# Data Source: fabric_datastore_vsphere
+## Example Usages
+This is an example of how to lookup vSphere fabric datastores.
+
+**Fabric vSphere data source by Id:**
+
+```hcl
+# Lookup vSphere fabric datastore using its name
+data "vra_fabric_datastore_vsphere" "this" {
+  id = var.fabric_datastore_vsphere_id
+}
```

Comment:
This is example to lookup vSphere fabric datastore using its id, not name. Please correct.